assets/app/data/crowbar/services/upgrade.factory.js

```diff
@@ -17,7 +17,8 @@
             createNewDatabaseServer: createNewDatabaseServer,
             connectDatabaseServer: connectDatabaseServer,
             cancelUpgrade: cancelUpgrade,
-            stopServices: stopServices
+            stopServices: stopServices,
+            upgradeNodes: upgradeNodes
```

Comment:
Add comma at the end. This minimizes the subsequent diffs and reduces conflicts.